lib/mix/tasks/ash_ai.gen.mcp.ex

```diff
@@ -116,7 +116,8 @@ if Code.ensure_loaded?(Igniter) do
                # You will need to set the `protocol_version_statement` to the
                # older version.
                protocol_version_statement: "2024-11-05",
-               otp_app: :#{otp_app}
+               otp_app: :#{otp_app},
+               path: "/mcp/ash"
```

Comment:
the new default should be `/ash_ai/mcp` I think based on our conversation, to line up w/ tidewave.